deployment/go.mod

```diff
@@ -0,0 +1,346 @@
+module github.com/smartcontractkit/chainlink/deployment
+
+go 1.22.8
+
+// Make sure we're working with the latest chainlink libs
+replace github.com/smartcontractkit/chainlink/v2 => ../
+
+require (
+	github.com/Masterminds/semver/v3 v3.2.1
+	github.com/avast/retry-go/v4 v4.6.0
+	github.com/aws/aws-sdk-go v1.54.19
+	github.com/ethereum/go-ethereum v1.13.8
+	github.com/hashicorp/consul/sdk v0.16.1
+	github.com/pkg/errors v0.9.1
+	github.com/smartcontractkit/ccip-owner-contracts v0.0.0-20240926212305-a6deabdfce86
+	github.com/smartcontractkit/chain-selectors v1.0.27
+	github.com/smartcontractkit/chainlink-ccip v0.0.0-20241022184834-e8564a286a47
+	github.com/smartcontractkit/chainlink-common v0.3.1-0.20241021103500-39a6e78c0286
+	github.com/smartcontractkit/chainlink-protos/job-distributor v0.4.0
+	github.com/smartcontractkit/chainlink/v2 v2.17.0
```

Comment:
If this is the lowest dep, then it is the one forcing the others to upgrade. Setting them all to the zero pseudo version at the same time should stick.

Reply:
Thanks!

Reply:
I think I've got it fixed.